File: Selenium/Activities/Activity10.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.common.action_chains import ActionChains

driver = webdriver.Chrome()
actions = ActionChains(driver)

driver.maximize_window()
driver.get("https://training-support.net/webelements/drag-drop")
print("Title of page is :", driver.title)

dropzone1 = driver.find_element(By.ID, "dropzone1")
dropzone2 = driver.find_element(By.ID, "dropzone2")
ball = driver.find_element(By.ID, "ball")

dropzone_text = driver.find_element(By.CLASS_NAME, "dropzone-text")

actions.click_and_hold(ball).move_to_element(dropzone1).release().perform()
if dropzone_text.text == "Dropped!":
    print("Ball is dropped in zone1")
else:
    print("Ball is not dropped in zone1")

actions.drag_and_drop(ball, dropzone2).perform()
if dropzone_text.text == "Dropped!":
    print("Ball is dropped in zone2")
else:
    print("Ball is not dropped in zone2")

driver.quit()

File: Selenium/Activities/Activity10.java
import org.openqa.selenium.By;
import org.openqa.selenium.Keys;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.chrome.ChromeDriver;
import org.openqa.selenium.interactions.Actions;

public class Activity10 {

    public static void main(String[] args) {

        ChromeDriver driver = new ChromeDriver();
        Actions action = new Actions(driver);
        driver.manage().window().maximize();

        driver.get("https://training-support.net/webelements/drag-drop");
        System.out.println("Title of page is : " + driver.getTitle());

        WebElement dropzone1 = driver.findElement(By.id("dropzone1"));
        WebElement dropzone2 = driver.findElement(By.id("dropzone2"));
        WebElement ball = driver.findElement(By.id("ball"));

        WebElement dropzoneText = driver.findElement(By.className("dropzone-text"));
        		
        action.clickAndHold(ball).moveToElement(dropzone1).release().build().perform();
        if(dropzoneText.getText().equals("Dropped!")) {
        	System.out.println("Ball is dropped in zone1");
        }
        else {
        	System.out.println("Ball is not dropped in zone1");
        }
        
        action.dragAndDrop(ball,dropzone2).build().perform();
        if(dropzoneText.getText().equals("Dropped!")) {
        	System.out.println("Ball is dropped in zone2");
        }
        else {
        	System.out.println("Ball is not dropped in zone2");
        }
        driver.quit();
    }
}
